components/rangeslider/rangesliders/rangeSliderBase.js

```diff
@@ -267,6 +288,19 @@ export default class RangeSliderBase {
         this.toggleThumb(value);
     }
 
+    /**
+     * @param {number} value
+     * @returns {void}
+     */
+    updatePolsNumberState(value) {
+        value = parseInt(value);
+        if (value < 2) value = 2;
+        if (!this.rangeslider.isStatePropValid('pols-number', value)) return;

```

Comment:
Do this check before the first one as the user can pass non numeric value and parsing it will return `Nan` that is a number and this check will pass however.
Also if it is NaN checking `NaN < 2` will return `false` and then the `value` won't be set to 2.

Check this example 
![image](https://github.com/CoherentLabs/GameUIComponents/assets/29014573/dae6022a-0f83-47ba-87cd-873485be28bf)

Reply:
However if the `polsNumber` is `NaN` it is well handled in the buildGrid method and it won't fail :D You are lucky or you have known that :D
![image](https://github.com/CoherentLabs/GameUIComponents/assets/29014573/dd854721-f39b-46fc-8d59-a810219e338d)
![image](https://github.com/CoherentLabs/GameUIComponents/assets/29014573/1165681f-9952-4ee9-a042-d2bae8243b52)

However, IMO let us not make it `NaN` because in the future there is a chance to use it in another code and we are going to be confused that it receives `NaN` as value :D